config/__init__.py

```diff
@@ -489,5 +486,9 @@ def get_update_feed() -> str:
     return 'https://raw.githubusercontent.com/EDCD/EDMarketConnector/releases/edmarketconnector.xml'
 
 
-# WARNING: update_feed is deprecated, and will be removed in 6.0 or later. Please migrate to get_update_feed()
```

Comment:
This appears to be incorrect. If I try and call update_feed, I receive the following error:

```
Exception in thread update worker:
Traceback (most recent call last):
  File "C:\Users\david\AppData\Local\Programs\Python\Python311\Lib\threading.py", line 1045, in _bootstrap_inner
    self.run()
  File "C:\Users\david\AppData\Local\Programs\Python\Python311\Lib\threading.py", line 982, in run
    self._target(*self._args, **self._kwargs)
  File "D:\Users\david\Documents\GitHub\EDMarketConnector\update.py", line 233, in worker
    newversion = self.check_appcast()
                 ^^^^^^^^^^^^^^^^^^^^
  File "D:\Users\david\Documents\GitHub\EDMarketConnector\update.py", line 184, in check_appcast
    request = requests.get(get_update_feed, timeout=10)
                           ^^^^^^^^^^^^^^^
NameError: name 'get_update_feed' is not defined. Did you mean: 'update_feed'?
```
This should instead throw a dep. warning.

Reply:
Looks like `get_update_feed` instead to me...

Reply:
Update, I am dumb.

Reply:
A canonical event for every developer out there: spend a long time chasing something that was caused by a very silly mistake that you somehow overlooked more than once.